README.md

```diff
@@ -0,0 +1 @@
+Super wspania�a aplikacja :)
```

Comment:
Co to za kod !!
Nie używaj polskich znaków.

Reply:
Polskie znaki są spoko